### PR TITLE
Adding "Grid Virtualization and Performance" subtopic.

### LINKED ISSUE #233: Create a separate igxGrid virtualization topic
Related to #228 

The topic should include the following: 
- Explanation when the grid will be virtualized
- Limitations to such grids
- What is required for virtualization, why width expected and enforced, why can't columns grow through templating outside the set width. 
- Examples for combinations of set width/heights along set widths of columns heights for rows
- Stackblitz link for the performance sample

### PR DESCRIPTION
**Pull request by MayaKirova**
Closes #233

### CODE REVIEW

**Comment by mpavlinov** [comment]
@MayaKirova As we support remote virtualization can you add a section about it as well.

**Comment by MayaKirova** [comment]
@mpavlinov We support it for the igxFor via additional event (onChunkPreload). At this time the grid exposes no similar event for accessing the chunk size and changing the rendering data.
If it’s something you feel needs to be implemented please add a new task/issue in the main repo so that we can add it to the spec and plan it accordingly.

**Comment by mpavlinov** [comment]
@MayaKirova Here is the link to the task in the main repo: https://github.com/IgniteUI/igniteui-angular/issues/877

File: en/components/grid_virtualization.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+title: Grid Virtualization and Performance
+_description: The Ignite UI for Angular Data Grid control now virtualizes its columns and rows, visualizing only the visible chunks of the data in the DOM, as a result allowing smooth and seamless scrolling through large sets of data.
+_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Angular Data Grid component, Angular Data Grid control, Angular Grid component, Angular Grid control, Angular High Performance Grid, Virtualization, Performance
+---
+
+## Grid Virtualization and Performance
+
+<p class="highlight">
+    Many modern applications require displaying big data in a way that is performant and seamless to the end user.
```
Comment (mpavlinov) [inline en/components/grid_virtualization.md]: @MayaKirova In this section we need to explain what virtualization is and how we use it in order to boost performance of the grid. Let me know if you want me to write this section.

Comment (MayaKirova) [reply en/components/grid_virtualization.md]: You can add a second paragraph with the information you deem neccessary.

**Pull request merged**